l10n_it_fatturapa_in/views/account_view.xml

```diff
@@ -425,6 +425,7 @@
                         <field name="fatturapa_payments" nolabel="1" readonly="1">
                             <tree string="Payments">
                                 <field name="payment_terms" />
+                                <field name="payment_methods" />
```

Comment:
Questo è un campo *2many quindi la visualizzazione così nella tree non mi pare molto utile perché viene visualizzato solo quanti record ci sono:
![image](https://user-images.githubusercontent.com/32064796/133643674-7a347a03-b6bf-4b9d-ba59-3e0f3a6ab885.png)
Potresti aggiungere il widget `many2many_tags` in modo da visualizzare qualche info più utile riguardo i record collegati?

Reply:
Giusto per capire, ha lo stesso problema anche la 12.0? https://github.com/OCA/l10n-italy/blob/349aaf23c81c763d46363c9b1f2d3fd93d9eed92/l10n_it_fatturapa_in/views/account_view.xml#L300

Reply:
Non l'ho provato ma suppongo di sì

Reply:
Il widget da solo non è molto utile perchè mostra dati tecnici:
![Screenshot from 2021-09-23 15-12-36](https://user-images.githubusercontent.com/1033131/134513754-4771410f-4f14-4020-89ac-49e83a8ecc1e.png)

Ho portato la modifica su 12 https://github.com/OCA/l10n-italy/pull/2433 aggiungendo `_rec_name`:
![Screenshot from 2021-09-23 15-13-25](https://user-images.githubusercontent.com/1033131/134513845-768fa5de-7bfb-477f-b15f-44146c06e343.png)